browser/main/Detail/SnippetNoteDetail.js

```diff
@@ -234,6 +234,26 @@ class SnippetNoteDetail extends React.Component {
     })
   }
 
+  handleTabDragStart (e, index) {
+    e.dataTransfer.setData('text/plain', index)
+  }
+
+  handleTabDrop (e, index) {
+    const oldIndex = parseInt(e.dataTransfer.getData('text'))
+
+    const snippets = this.state.note.snippets.slice()
+    const draggedSnippet = snippets[oldIndex]
+    snippets[oldIndex] = snippets[index]
+    snippets[index] = draggedSnippet
+    const snippetIndex = index
```

Comment:
Could you fix a bug shown in https://gyazo.com/033ea6323d18720dbc108481ca70e6fc ?
It fixes after reload boostnote, so `this.state.note.snippets` will be wrong after drop I think.

Reply:
Hi @mslourens

Reply:
sorry for my late response @kazup01, I was very busy lately. I fixed the bug, I forgot to reload the tab at the oldIndex. Now it's working again.

Reply:
That's ok, thanks for your support ;)

Reply:
Could you review it again? @kohei-takata